lang/java/reef-tang/tang/src/test/java/org/apache/reef/tang/ClassHierarchyDeserializationTest.java

```diff
@@ -35,16 +35,21 @@
 
 /**
  * Test case for class hierarchy deserialization.
+ * TODO: The files should be created and deserialized by the AvroClassHierarchySerializer (REEF-400)
  */
 public class ClassHierarchyDeserializationTest {
+  private final ConfigurationSerializer configurationSerializer = new AvroConfigurationSerializer();
+  private final ClassHierarchySerializer classHierarchySerializer = new AvroClassHierarchySerializer();
 
   /**
    * generate task.bin from running .Net ClassHierarchyBuilder.exe
    */
   @Test
   public void testDeserializationForTasks() {
+    // TODO: The file should be written by Avro (REEF-400)
```

Comment:
Sure. Once C# side is done, I will regenerate those files and push them in, may be give another name.  The best way is to generate on the fly. But we need to figure out the dependency.